src/commands/sync_leaguepedia.py

```
@@ -119,21 +144,133 @@ async def _process_contest(
+                await ctx.db_session.flush()
+            except Exception:
+                # If flush fails, log and skip notification for safety
+                logger.exception(
+                    "Failed to flush session after saving result for match %s",
+                    match.id,
+                )
+                return None
+
+            # Record IDs for later notification after commit
+            ctx.notifications.append((match.id, result.id))
+            return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)